roles/openshift_master_upgrade/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: your name
+  description: OpenShift Master upgrade
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 2.1
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- role: openshift_common
```

Comment:
That is ok, that works as expected. Just, if you have two plays each one including the same role, dependency of that role gets run twice, once for each play. I need to check the individual ``openshift*_facts`` roles to see how they work.